index/src/graph/chain.rs

```diff
@@ -48,6 +48,7 @@ pub struct Chain {
     // Encoded native asset location for Sub-chains
     pub native_asset: Vec<u8>,
     pub foreign_asset: Option<ForeignAssetModule>,
+    pub storage_handler: Vec<u8>,
```

Comment:
Why call it `storage_handler`? I think `handler` is good to explain its meaning

Reply:
"handler" is confusing, give it a concrete name, I named it storage_handler because the functionality of the contract with that address is to store solutions, right?